src/HomePage/HomePage.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import Header from "../Header/Header";
+import Footer from "../Footer/Footer";
+import MovieCard from "../MovieCard/MovieCard";
+import movieData from "../data/movies.js";
+import "./HomePage.css";
+
+
+class HomePage extends Component {
+  constructor() {
+    super()
+    this.state= {
+      movies: movieData.movies,
+    }
+  }
+  
+  displayMovies = () => {
+    const movieList = this.state.movies.map(movie => {
+      return <MovieCard 
+        key={movie.id} 
+        id={movie.id} 
+        posterImage={movie.poster_path} 
+        title={movie.title} 
+        averageRating={movie.average_rating.toFixed(1)} 
+        releaseDate={movie.release_date}/>
+    })
+    return movieList; 
```

Comment:
Looks like we are be destructing the same info in both the displayMovies and MovieCard. This may want to consider this for refactoring.

Reply:
Here we are passing in the prop names and then in the MovieCard component instead of using "props" we are destructuring and passing in our prop name directly. 

We could try to pass a prop such as "<MovieCard movieInfo={movie}/>" and then in the MovieCard component, we could destructure like: 
{ movie.title, movie.poster_path...etc} instead of passing props and having to do: 
props.movie.title, props.movie.poster_path...etc

I'm not sure if that is the way to go, but we can try it and see if we like it?

Reply:
Let leave it for now and revisit at a later time